matRad_calcDoseDirect.m

```diff
@@ -71,50 +75,85 @@
     end
 end
 
+for i = 1:(length(stf))
+    beamInfo(i).suffix = ['Beam', num2str(i)];
+end
+
 % compute phyical dose
-resultGUI.physicalDose = reshape(full(dij.physicalDose{1}(:,1)),ct.cubeDim);
+sumPhysical = zeros(ct.cubeDim);
+for i = 1:length(beamInfo)
+    resultGUI.(['physicalDose', beamInfo(i).suffix]) = reshape(full(dij{i}.physicalDose{1}(:,1)),ct.cubeDim);
```

Comment:
This line of code is not necessary. You can simply write 

sumPhysical(:) = sumPhysical(:) + full(dij{i}.physicalDose{1}(:,1));

below - and I dont even know if you need the full command for the addition...